podium-uaa/src/test/resources/config/application.yml

```diff
@@ -99,3 +99,7 @@ podium:
         contactEmail:
         license:
         licenseUrl:
+
+uaa:
+    security:
+        accountLockingPeriodSeconds: 3
```

Comment:
It is, however if it isnt mentioned in the properties file then its not clear that that value is configurable